server/index.ts

```diff
@@ -12,17 +12,17 @@ app
   .get("/user", async (_, res) => {
     try {
       const user = await getUser();
-      res.send(user).status(200);
+      res.status(200).send(user);
     } catch (e) {
-      res.send("Whoops!").status(500);
+      res.status(500).send("Whoops!");
```

Comment:
Trigger the `.status()` callback after chaining the `.send()` callback won't work. In order to properly send the HTTP statuses, we have to set the `status` code prior sending it out.